### PR TITLE
Fix for DYN-4011-Popups-Border Bug

### PR DESCRIPTION
**Pull request by RobertGlobant20**
### Purpose

I updated the PopupWindow in order to add a shadow in the border of the window and the popup pointer also the Popup location and size need to be updated by 10 pixels in the top and bottom so we can show the shadow.
The shadow has the next features:
- Color Black 0xFFFFFF
- Transparency 20%
- Shadow of 4px lenght

### Declarations

Check these if you believe they are true

- [X] The codebase is in a better state after this PR
- [X] Is documented according to the [standards](https://github.com/DynamoDS/Dynamo/wiki/Coding-Standards)
- [X] The level of testing this PR includes is appropriate
- [X] User facing strings, if any, are extracted into `*.resx` files
- [ ] All tests pass using the self-service CI.
- [ ] Snapshot of UI changes, if any.
- [ ] Changes to the API follow [Semantic Versioning](https://github.com/DynamoDS/Dynamo/wiki/Dynamo-Versions) and are documented in the [API Changes](https://github.com/DynamoDS/Dynamo/wiki/API-Changes) document.
- [ ] This PR modifies some build requirements and the readme is updated

### Reviewers

@QilongTang 

### FYIs

@filipeotero

### CODE REVIEW

File: src/DynamoCoreWpf/Views/GuidedTour/PopupWindow.xaml.cs [hunk 1]
```diff
@@ -23,10 +23,13 @@ public PopupWindow(PopupWindowViewModel viewModel, HostControlInfo hInfo)
 
             //Due that we are drawing the Direction pointers on left and right side of the Canvas (10 width each one) then we need to add 20
             RootLayout.Width = popupViewModel.Width + 20; 
-            RootLayout.Height = popupViewModel.Height;
+            //Also a shadow of 10 pixels in top and 10 pixels at the bottom will be shown then we need to add 20
+            RootLayout.Height = popupViewModel.Height + 20;
```
Comment (QilongTang) [inline src/DynamoCoreWpf/Views/GuidedTour/PopupWindow.xaml.cs:27]: Let's define this 10 in the  class and use 10 * 2 here, maybe there will be clearer

File: src/DynamoCoreWpf/Views/GuidedTour/PopupWindow.xaml [hunk 2]
```diff
@@ -27,19 +27,52 @@
...
+                <DropShadowEffect Opacity="0.2" 
+                                  Color="Black"
+                                  ShadowDepth="4"
+                                  BlurRadius="4"
+                                  Direction="315"/>
+            </Path.Effect>
+        </Path>
+
+
+        <Polygon x:Name="TooltipPointer"
```
Comment (QilongTang) [inline src/DynamoCoreWpf/Views/GuidedTour/PopupWindow.xaml:63]: Some comments to this block would be appreciated!

**Review by QilongTang — COMMENTED** [review]
Some comments, would you also attach the screenshots of UI diff?

File: src/DynamoCoreWpf/Views/GuidedTour/PopupWindow.xaml.cs (hunk 1 above)
Comment (RobertGlobant20) [reply src/DynamoCoreWpf/Views/GuidedTour/PopupWindow.xaml.cs]: @QilongTang fixed in the next commit: https://github.com/DynamoDS/Dynamo/pull/12048/commits/c84d82b2d65d9ce4ea1dcf90649974be6f4618cd

File: src/DynamoCoreWpf/Views/GuidedTour/PopupWindow.xaml (hunk 2 above)
Comment (RobertGlobant20) [reply src/DynamoCoreWpf/Views/GuidedTour/PopupWindow.xaml:63]: @QilongTang I added extra comments in the next commit:
https://github.com/DynamoDS/Dynamo/pull/12048/commits/c84d82b2d65d9ce4ea1dcf90649974be6f4618cd

**Comment by RobertGlobant20** [comment]
Screenshot of the UI Diff for the shorcuts popup.
popup_toolbar_screenshot

**Comment by RobertGlobant20** [comment]
GIF of the shadow in the shortcuts popup
shadow_popups_tour

**Review by QilongTang — APPROVED** [review]
LGTM

**Pull request merged**